code/modules/materials/Mat_Materials.dm

```diff
@@ -383,10 +384,10 @@
 	New()
 		value = 300
 
-		setProperty("density", 2)
+		setProperty("density", 45)
 		setProperty("hard", 2)
 		setProperty("reflective", 55)
-		setProperty("electrical", 60)
+		setProperty("electrical", 75)
```

Comment:
Electrum, which is supposed to be a top-tier conductor, has an `electrical` prop of 80. This is a little too close to that- either bump it down to ~70 or buff electrum to ~90 (lazy estimates, feel free to choose whatever as long as it makes sense)

Reply:
Id say nerf it for one very specific powergame strat, one of the gimmick jobs spawns with both a stun cane and 3 gold bullions, so this would make it really easy to get a good battery for the stun cane really early on.

Reply:
currently 75 is the exact same conductivity as claretine has, which is far easier to obtain than gold and much more common in ores.  I'll put it down to 72 to make it slightly worse since it's both rarer than claretine and very conductive IRL, and I don't foresee the one specific use of all the gold for the gimmick job being an issue.

Reply:
I'd rather just buff it, matsci has been pretty stagnant for a long time. The CEO line of jobs do spawn with 2 bullions and a cane, unless that's been changed to 3. It's not that hard to get a stun baton, so i don't think going through the effort of making a better battery is that game breaking vs. just blowing up a sec crate and doing it. Especially since QMs are going to bend over backwards to get the bullions from the CEO, they'd totally order a sec crate lmao.

Reply:
The issue is those gold bullions can be smelted down into gold, which allows the stun cane to become a better stun baton, no cargo and bombs required. Hence why id say nerf gold, itd allow ceos and the like to get stun canes with good batteries very quickly